### PR TITLE
Feature/single movie

### PR DESCRIPTION
**Pull request by Afoucheaux**
## Purpose
To implement the User Story of iteration 2.
Feature single move page.
## Approach
Paired programming,  class note and React doc's.
#### Open Questions and Pre-Merge TODOs
- [x] Check for syntax errors.
- [x] Look for refactoring possibilities. 
- [x] Note any other concerns. 
## Learning
Better understanding of React

### CODE REVIEW

File: src/MovieSnapShot/MovieSnapShot.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      singleMovie: singleMovie.movie
+    }
+  }
+
+  render() {
+    const genres = this.state.singleMovie.genres.join(" | ");
+    return (
+      <div className="background-img" style={{backgroundImage: `url(${this.state.singleMovie.backdrop_path})`}}>
+        <Header onHomeClick={this.props.onHomeClick}/>
+        <main className="single-movie-container">
```
Comment (elizhahn) [inline src/MovieSnapShot/MovieSnapShot.js:21]: I am still wondering if we can declare two main tags even if they are in separate components that render conditionally. Something we should ask about I think, I couldn't find any articles relating to it when doing research

File: src/App/App.js [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
+    this.setState({ clickedMovieId: "" })
+  }
+
+  render() {
+    return (
+      <>
+        {!this.state.clickedMovieId ?
+        <HomePage onMovieClick={this.onMovieClick}/> : <MovieSnapShot onHomeClick={this.onHomeClick}/>}
+      </>
+    )
```
Comment (elizhahn) [inline src/App/App.js:27]: Could think about using a variable here for the conditional render for readability: 

const userView = !this.state.clickedMovieId ? `<HomePage onMovieClick={this.onMovieClick}/>` : `<MovieSnapShot onHomeClick={this.onHomeClick}/>`

return (
<>
  {userView}
</>
)

**Review by elizhahn — APPROVED** [review]

File: src/MovieSnapShot/MovieSnapShot.js (hunk 1 above)
Comment (Afoucheaux) [reply src/MovieSnapShot/MovieSnapShot.js:21]: I will ask my mentor and see if he can point us down the right path. If not we should reach out to Leta or Scott.

File: src/App/App.js (hunk 2 above)
Comment (Afoucheaux) [reply src/App/App.js:27]: When refactoring this weekend we can make that change if that the direction we want to go.

File: src/MovieSnapShot/MovieSnapShot.js (hunk 1 above)
Comment (elizhahn) [reply src/MovieSnapShot/MovieSnapShot.js:21]: Ok, same here with my mentor

**Pull request merged**